tools/docker/run/Dockerfile

```diff
@@ -19,11 +24,14 @@ RUN sudo bash -c "DEBIAN_FRONTEND=noninteractive apt-get install --yes --allow-u
 
 # Now a fresh non-sudo user:
 
-RUN sudo  bash -c "\
+RUN sudo bash -c "\
   adduser --uid 20042 --disabled-password --gecos '' biokepi && \
   passwd -l biokepi && \
   chown -R biokepi:biokepi /home/biokepi"
 USER biokepi
 ENV HOME /home/biokepi
 WORKDIR /home/biokepi
 RUN opam init -a --yes --comp 4.02.3
+
+COPY fonts.conf $HOME/.config/fontconfig/fonts.conf
+RUN ln -s $HOME/.config/fontconfig/fonts.conf $HOME/.fonts.conf
```

Comment:
(Just out of curiosity)  is there a know way to make this setup system-wide?

That would make it a bit less fragile (?).